cmds/lddfiles/lddfiles.go

```diff
@@ -1,4 +1,4 @@
-// Copyright 2009-2017 The Go Authors.  All rights reserved.
```

Comment:
@rminnich You added this file. Are you sure you didn't mean to write "u-root Authors"?

Reply:
wow, yeah, what was I thinking.